src/multibody/liegroup/fwd.hpp

```diff
@@ -23,6 +23,15 @@
 namespace se3
 {
   template<typename LieGroupCollection> struct LieGroupGenericTpl;
+
```

Comment:
Maybe this can be in the fwd.hpp of the project. 
With an enum of the type
```
enum ArgumentPosition
{
  arg0 = 0,
  arg1 = 1,
  ...
}
```
@jmirabel What do you think?

Reply:
But then `dDifference<ARG0>`, `dDifference_d<ARG0>` or `dDifferenceD<ARG0>` ?

Reply:
I would say `dDifference<ARG0>` which is explicit enough. What do you think?

Reply:
Let's go for `dDifference<ARG0>`